mmv1/products/cloudidentity/api.yaml

```diff
@@ -156,10 +156,75 @@ objects:
           Google Groups are the default type of group and have a label with a key of cloudidentity.googleapis.com/groups.discussion_forum and an empty value.
 
           Existing Google Groups can have an additional label with a key of cloudidentity.googleapis.com/groups.security and an empty value added to them. This is an immutable change and the security label cannot be removed once added.
-          
+
           Dynamic groups have a label with a key of cloudidentity.googleapis.com/groups.dynamic.
-          
+
           Identity-mapped groups for Cloud Search have a label with a key of system/groups/external and an empty value.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'dynamicGroupMetadata'
+        required: false
+        input: true
+        description: |
+          Dynamic group metadata like queries and status.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'queries'
+            description: |
+              Memberships will be the union of all queries. Only one entry with USER resource is currently supported. Customers can create up to 100 dynamic groups.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+                - !ruby/object:Api::Type::Enum
+                  name: 'resourceType'
+                  description: |
+                    Resource type for the Dynamic Group Query
+                  values:
+                    - :RESOURCE_TYPE_UNSPECIFIED
```

Comment:
```suggestion
```

Reply:
FIELDNAME_UNSPECIFIED values are an artifact of how Google APIs are produced, they're the enum representation of null. They're never in the valid range of values!

Reply:
Thanks! I've removed this.